AppHandling/Run-TestsInNavContainer.ps1

```diff
@@ -99,6 +106,8 @@ function Run-TestsInBcContainer {
         [Parameter(Mandatory=$false)]
         [string] $testCodeunit = "*",
         [Parameter(Mandatory=$false)]
+        [string] $testCodeunitRange = "*",

```

Comment:
Default should not be "*" - default should be empty or null to make sure that the code doesn't run and won't change any behavior from earlier.

Reply:
Done.